test/ctest_helpers.cmake

```diff
@@ -77,6 +77,24 @@ function(create_test_executable target)
         target_link_libraries(${target} ${CREATE_TEST_LIBS})
     endif()
 
+    if(WT_LINUX OR WT_DARWIN)
+        # Link the final test executable with a relative runpath to the
+        # top-level build directory. This being the build location of the
+        # WiredTiger library.
+        set(origin_linker_variable)
+        if(WT_LINUX)
+            set(origin_linker_variable "\\$ORIGIN")
+        elseif(WT_DARWIN)
+            set(origin_linker_variable "@loader_path")
+        endif()
+        file(RELATIVE_PATH relative_rpath ${test_binary_dir} ${CMAKE_BINARY_DIR})
+        set_target_properties(${target}
+            PROPERTIES
+                BUILD_WITH_INSTALL_RPATH FALSE
```

Comment:
Please add a comment on why this value is set to `FALSE` as it might be useful for anyone who reads this code.

Reply:
Added in [e02da67](https://github.com/wiredtiger/wiredtiger/pull/7524/commits/e02da67e0aebe0cf8031d83aaaa0fa69a9be46e5)